cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -159,7 +159,7 @@ def remove_deleted_items():
             error_list.append(_('General indexing error occurred'))
 
         if error_list:
-            raise SearchIndexingError(_('Error(s) present during indexing'), error_list)
+            raise SearchIndexingError('Error(s) present during indexing', error_list)
```

Comment:
@sarina I think you're right. This string is not displayed to user. So remove it from i18n list.